templates/providerstoragecluster.go

```diff
@@ -95,6 +95,9 @@ var ProviderStorageClusterTemplate = ocsv1.StorageCluster{
 			CephFilesystems: ocsv1.ManageCephFilesystems{
 				DisableStorageClass: false,
 			},
+			CephConfig: ocsv1.ManageCephConfig{
+				ReconcileStrategy: "ignore",
```

Comment:
I am not sure about the side effects of this.But this need to be there.  what do you think @nb-ohad ?